jwst/background/tests/test_background.py

```diff
@@ -45,6 +45,9 @@ def background(tmpdir_factory):
         image.meta.subarray.xstart = 1
         image.meta.subarray.ystart = 1
 
+        image.meta.subarray.xsize = image.data.shape[-1]
```

Comment:
Need to also add these lines in the `science_image` function, down around line 75 in this module.

Reply:
whoops! sorted